### PR TITLE
Fix tracing param in disjoint_pool_parser

### PR DESCRIPTION
**Pull request by igchor**
The disjoint_pool implementation distinguishes between a few different levels of logging (0-3). The argument should be an int, not bool.

### CODE REVIEW

**Review by pbalcer — COMMENTED** [review]
how did nothing catch this? if anything but 0/1/false/true was assigned to this variable, coverity would have complained, no?

File: source/adapters/level_zero/usm.cpp [hunk 1]
```diff
@@ -64,7 +64,7 @@ usm::DisjointPoolAllConfigs InitializeDisjointPoolConfig() {
                                  ? PoolUrTraceVal
                                  : (PoolPiTraceVal ? PoolPiTraceVal : nullptr);
 
-  bool PoolTrace = false;
+  int PoolTrace = false;
```
[the remark this answers is not in the document]
Comment (igchor) [reply source/adapters/level_zero/usm.cpp]: Done.

**Comment by igchor** [comment]
> how did nothing catch this? if anything but 0/1/false/true was assigned to this variable, coverity would have complained, no?

Yeah, I would think so, or that there should be a warning here. But for a warning, it seems it would only be generated if we'd use braced initialization.

**Review by pbalcer — APPROVED** [review]

**Comment by igchor** [comment]
I just realized there was one more problem. I fixed it in a separate commit.

**Comment by DamianDuy** [comment]
LGTM

**Pull request merged**